RxExample/RxExample/Examples/GitHubSearchRepositories/GitHubSearchRepositoriesViewController.swift

```diff
@@ -43,8 +43,9 @@ class GitHubSearchRepositoriesViewController: ViewController, UITableViewDelegat
 
 
```

Comment:
I think we can define local variable table here.

`let tableView = self.tableView`